filter/etc/e-smith/templates/etc/rspamd/rspamd.local.lua/10base

```diff
@@ -0,0 +1,15 @@
+{
+    if ( $rspamd{'SpamCheckStatus'} eq 'enabled') {
+        $OUT .= "\n -- symbols/lua rules are not disabled\n";
+    } else {
+        # Disable all lua rules
+        $OUT .= "-- disable symbol rules\n";
+
+        my @symbols = qw(
+            SPOOF_REPLYTO
+        );
```

Comment:
If you move these lines in a separate template fragment (say, `00template_vars`) and define the array as `our @symbols` it would be easier to mangle it with a custom template

```suggestion
```

Reply:
done